widget/menu_item.go

```diff
@@ -15,14 +15,33 @@ var _ fyne.Widget = (*menuItem)(nil)
 // menuItem is a widget for displaying a fyne.menuItem.
 type menuItem struct {
 	widget.Base
-	Item   *fyne.MenuItem
-	Parent *Menu
-
+	Item            *fyne.MenuItem
+	Parent          *Menu
 	child           *Menu
 	hovered         bool
+	focused         bool
 	onActivateChild func(*menuItem)
 }
 
+// FocusGained is called when the Entry has been given focus.
+func (i *menuItem) FocusGained() {
+	i.focused = true
+	i.hovered = true
```

Comment:
I don't think hovered should be set on focus - it is a different status

Reply:
I agree